app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <div class="checkbox">
+        <%= form.label :template do %>
+          <%= form.check_box :template %>
+          This stage can serve as a production template (maximum of 1 per project).
```

Comment:
ehhh. All the stages can be cloned. The point about this is that this particular stage *should* be cloned.